pkg/types/dsse/dsse.go

```diff
@@ -75,86 +76,112 @@ func ToLogEntry(ds *pb.DSSERequestV0_0_2, algorithmRegistry *signature.Algorithm
 			return nil, fmt.Errorf("must contain either a public key or X.509 certificate")
 		}
 	}
-	signerVerifiers, err := verifyEnvelope(verifiers, ds.Envelope, algorithmRegistry)
+	signerVerifiers, dsseHashAlgs, err := verifyEnvelope(verifiers, ds.Envelope, algorithmRegistry)
 	if err != nil {
 		return nil, err
 	}
-	payloadHash := sha256.Sum256(ds.Envelope.Payload)
+
+	// Canonicalize the order of the signatures
```

Comment:
I think it would be helpful to define canonicalization algorithm here.

Reply:
Updated in https://github.com/sigstore/rekor-tiles/pull/220/